### PR TITLE
docs: improve balloons documentation

### PR DESCRIPTION
**Pull request by askervin**
- Describe how the policy works.
- Document relevant configuration parameters.
- Add CPU classes.

### CODE REVIEW

File: docs/policy/balloons.md [hunk 1]
```diff
@@ -23,25 +57,124 @@ system of CRI-RM. See [setup and
...
+    balloons. If there are balloon types with pre-created balloon
+    instances (`MinBalloons` > 0), balloons with the highest
+    `AllocatorPriority` are created first.
+
+Related configuration parameters:
+- `policy.ReservedResources.CPU` specifies the (number of) CPUs in the
+  special `reserved` balloon. By default all containers in the
+  `kube-system` namespace are assigned to the reserved balloon.
+- `cpu.classes` defines CPU classes and their parameters (such as `minFreq`
+  and `maxFreq`).
```
[the remark this answers is not in the document]
Comment (askervin) [reply docs/policy/balloons.md]: Added.

File: docs/policy/balloons.md [hunk 2]
```diff
@@ -23,25 +57,124 @@ system of CRI-RM. See [setup and
...
+  classes:
+    lowpower:
+      minFreq: 800
+      maxFreq: 800
+    dynamic:
+      minFreq: 800
+      maxFreq: 3600
+    turbo:
+      minFreq: 3000
+      maxFreq: 3600
```
[the remark this answers is not in the document]
Comment (askervin) [reply docs/policy/balloons.md:152]: Added.

File: docs/policy/balloons.md [hunk 3]
```diff
@@ -3,11 +3,45 @@
...
-configuring balloon-specific CPU classes.
+are disjoint CPU pools. Balloons can be inflated and deflated, that is
+CPUs added and removed, based on the CPU resource requests of
+containers. Balloons can be static or dynamically created and
+destroyed. CPUs in balloons can be configured, for example, by setting
+min and max frequencies.
+
+## How It Works
+
+1. A balloon is a set of CPUs.
```
[the remark this answers is not in the document]
Comment (askervin) [reply docs/policy/balloons.md]: Thanks, this is a great point.

I ended up using _balloon_ and _balloon type_, where the first always refers to instances of a balloon type. And I added new first point to this section to hopefully make this terminology clear.

[the remark this answers is not in the document]
Comment (marquiz) [reply docs/policy/balloons.md]: Looks good 👌

**Review by marquiz — APPROVED** [review]
Thanks @askervin for this, looks good
LGTM

**Pull request merged**